Sources/TiqrCoreObjC/Classes/TiqrCoreManager.m

```diff
@@ -60,34 +60,10 @@ + (instancetype)sharedInstance {
         sharedInstance = [[self alloc] init];
     
         UIStoryboard *storyboard = [UIStoryboard storyboardWithName:@"Main" bundle:SWIFTPM_MODULE_BUNDLE];
+//        StartViewController *startViewController = [storyboard instantiateInitialViewController];
```

Comment:
Why is this commented out?

Reply:
ah goeie, daar had ik even geprobeerd om een deel van wat er in het storyboard wordt gedaan in code te doen, maar ik heb het niet gebruikt uiteindelijk, ik zal het weer weg gooien